docs/contributing/coding-style.md

```diff
@@ -517,7 +535,13 @@ func New(opts ...Option) (Server, error) {
     srv := new(server)
     for _, opt := range opts {
         if err := opt(srv); err != nil {
-            return nil, errors.ErrOptionFailed(err, reflect.ValueOf(opt))
+            werr := errors.ErrOptionFailed(err, reflect.ValueOf(opt))
+
```

Comment:
```suggestion
```
```suggestion

```

Reply:
I think it is better to keep this line break to differentiate `werr` and `err`